data-migration/src/index.ts

```diff
@@ -0,0 +1,3 @@
+import { migrateData } from './migration/migrate';
+
+migrateData();
```

Comment:
this would be a good place to take `process.argv` params, so as not to rely on env vars (which some people with restricted permissions wouldn't be able to set)